### PR TITLE
BZ1883750: Vmware VM - disable hibernation (CNV 2.5)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1883750

CP to 4.6 and 4.7

### CODE REVIEW

**Comment by apinnick** [comment]
Verified by QE. Ready for merge

File: modules/virt-importing-vm-prerequisites.adoc [hunk 1]
```diff
@@ -41,9 +41,9 @@ endif::[]
 ifdef::virt-importing-vmware-vm[]
 Importing a VMware VM into {VirtProductName} has the following prerequisites:
 
-* You must have access to an image registry.
-* You must create a VMware Virtual Disk Development Kit (VDDK) image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
-* You must power off the VMware virtual machine before you import it.
+* You must create a VMware Virtual Disk Development Kit (VDDK) image, push it to an image registry that is accessible to your {VirtProductName} environment, and add it to the `v2v-vmware` ConfigMap.
+* If you importing a VM with a Windows operating system, you must disable hibernation.
```
Comment (bergerhoffer) [inline modules/virt-importing-vm-prerequisites.adoc:45]: "If you importing" -> "If you are importing"

**Review by bergerhoffer — COMMENTED** [review]
Just one missing word

File: modules/virt-importing-vm-prerequisites.adoc (hunk 1 above)
Comment (apinnick) [reply modules/virt-importing-vm-prerequisites.adoc]: Fixed

**Pull request merged**